docs/public/specifications/test-zones/Address-TP/address01.md

```diff
@@ -0,0 +1,197 @@
+irrelevant to the test scenario and must be ignored.
+
+Scenario name        | Mandatory message tag                                                        | Forbidden message tags
+:--------------------|:-----------------------------------------------------------------------------|:-------
+GOOD-1               | A01_ADDR_GLOBALLY_REACHABLE                                                  | 2) 
+ALL-NON-REACHABLE    | A01_NO_GLOBALLY_REACHABLE_ADDR                                               | 2) 
+MIXED-LOCAL-DOC-1    | A01_LOCAL_USE_ADDR, A01_DOCUMENTATION_ADDR                                   | 2) 
+MIXED-LOCAL-DOC-2    | A01_LOCAL_USE_ADDR, A01_DOCUMENTATION_ADDR                                   | 2) 
+MIXED-LOCAL-OTHER-1  | A01_LOCAL_USE_ADDR, A01_ADDR_NOT_GLOBALLY_REACHABLE                          | 2) 
+MIXED-LOCAL-OTHER-2  | A01_LOCAL_USE_ADDR, A01_ADDR_NOT_GLOBALLY_REACHABLE                          | 2) 
+MIXED-DOC-OTHER-1    | A01_DOCUMENTATION_ADDR, A01_ADDR_NOT_GLOBALLY_REACHABLE                      | 2) 
+MIXED-DOC-OTHER-2    | A01_DOCUMENTATION_ADDR, A01_ADDR_NOT_GLOBALLY_REACHABLE                      | 2) 
+MIXED-ALL-1          | A01_ADDR_NOT_GLOBALLY_REACHABLE, A01_DOCUMENTATION_ADDR, A01_LOCAL_USE_ADDR  | 2) 
+MIXED-ALL-2          | A01_ADDR_NOT_GLOBALLY_REACHABLE, A01_DOCUMENTATION_ADDR, A01_LOCAL_USE_ADDR  | 2) 
+
+* (1) All tags except for those specified as "Forbidden message tags" (no instances for these test scenarios)
+* (2) All tags except for those specified as "Mandatory message tags"
+
+## Zone setup for test scenarios
+
+Assumptions for the scenario specifications unless otherwise specified for
+the specific scenario:
+* The child zone is `SCENARIO.address01.xa`.
+* There is no zone file or zone data for the child zone.
+* For each scenario zone there are two NS records (ns[1-2]).
+  * All NS are in-bailiwick
+  * All NS have both IPv4 and IPv6 addresses
+  * All required glue are present in the delegation.
+* EDNS, version 0, is included in all responses on queries with EDNS.
+* EDNS is not included in responses on queries without EDNS.
+
+Address designation      | Meaning                                                        
+:------------------------|:------------------------------------------------------------------------------------------------
+OK                       | Globally routable, public address
+LOCAL_USE_ADDR           | Address part of range used for private networks (loopback, RFC1918, Provider shared, etc.)
+DOCUMENTATION_ADDR       | Address part of range used for documentation purposes.
+NOT_GLOBALLY_REACHABLE   | Address part of any other range listed as not globally reachable
+
+Designations are based on the address block ranges from the 
+[Special purpose IPv4 addresses] and [Special purpose IPv6 addresses] registries. 
+
+### GOOD-1 
+The "happy path". Everything is fine.
+
+* Zone: good-1.address01.xa
+
+### MIXED-LOCAL-DOC-1
+
+* Zone mixed-local-doc-1.address01.xa
```

Comment:
I suggest "Zone:" to make it easier to see that it is a statement of the zone name.